src/test/java/uk/gov/justice/laa/crime/cfecrime/steps/RequestHandlerStepDefs.java

```diff
@@ -28,43 +31,49 @@ public Boolean bool(String value){
         return Boolean.valueOf(value);
     }
 
-    private Result.Outcome oc = null;
+    private Outcome outcome = null;
     @Given("Client Under Eighteen {string} Passport benefited {string}")
-    public void client_under_eighteen_passport_benefited(String under18, String passportedBenefit) {
+    public void client_under_eighteen_passport_benefited(String under18, String passportedBenefit) throws UndefinedOutcomeException, JsonProcessingException {
        if (Boolean.valueOf(under18)) {
            RequestTestUtil.setSectionUnder18(request, true);
        }
 
         if (Boolean.valueOf(passportedBenefit)) {
             RequestTestUtil.setSectionPassportBenefit(request, true);
         }
-
+        String jsonString = RequestTestUtil.getRequestAsJson(request);
+        log.info("request = "+ jsonString);
```

Comment:
this jsonString doesn't seem to used?

Reply:
I will think about this

Reply:
output json request string when exception is caught